tests/utils/cassandra/cassandra.go

```diff
@@ -120,3 +122,19 @@ func getConfigurationFromNodeLogs(
 
 	return configuration, nil
 }
+
+// Cqlsh TODO function comment.
```

Comment:
add the comment?

Reply:
added in https://github.com/mesosphere/kudo-cassandra-operator/pull/31/commits/a2162f4231fbebb5b27abc09ffebed78bbc0b93c